src/manifest/manifest.ts

```diff
@@ -85,6 +85,7 @@ interface IManifestParsingOptions {
   /* eslint-enable import/no-deprecated */
   /** External callback peforming an automatic filtering of wanted Representations. */
   representationFilter? : IRepresentationFilter;
+  manifestUpdateUrl? : string;
```

Comment:
Maybe we should add a comment also there.
Also we may want to specify what will happen if this is `undefined`

Reply:
Also maybe we should explicitely link somewhere the notion of this update URL with `MANIFEST_UPDATE_TYPE.Partial` because that's kind of implicit here.

(Here? In `MANIFEST_UPDATE_TYPE` doc? At both places?)

Reply:
Yeah i have been thinking about it when making the PR. The partial update type implies that we use the update URL.
I'll think about it.